firefox-ios/Client/Frontend/Browser/StartAtHomeHelper.swift

```diff
@@ -44,27 +44,24 @@ class StartAtHomeHelper: FeatureFlaggable {
     /// the user's last activity and whether, based on their settings, Start at Home feature
     /// should perform its function.
     public func shouldStartAtHome() -> Bool {
-        let setting = startAtHomeSetting
-        guard setting != .disabled else { return false }
+            let setting = startAtHomeSetting
```

Comment:
the indenting here looks off. You might want to select tho whole file and reindent.